src/uct/ib/rdmacm/ucx-rdmacm.pc.in

```diff
@@ -0,0 +1,5 @@
+Name: @PACKAGE@-rdmacm
+Description:  Unified Communication X Library RDMACM module
```

Comment:
extra space before Unified (in few places)

Reply:
removed extra space